speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
```

Comment:
I was just confused about what Amazon means by "extending the intent."  I still don't really know, but if it works.. it's fine.